docs/100-upgrade/migration-guides/3.3-3.4.mdx

```diff
@@ -520,6 +520,52 @@ must not be versioned:
 
 ## Code changes
 
+### `entry.worker.ts`
```

Comment:
TO be merged within the section introduced by @paul-vd in https://github.com/front-commerce/developers.front-commerce.com/pull/886/files#diff-5988c547fcac98c09dc7779e885fae823a4ed249bb4fe0b3525397234c10b458R902